### PR TITLE
Update tokenizer in evaluation

### PR DESCRIPTION
**Pull request by manandey**
-> Updated the tokenizer (hardcoded as of now, and would need to change the repo and subfolder if we change it while training again).
-> Commented the code where we were setting the logits of entity related special tokens to -inf as of now until the issue of 'nan's' is fixed.

### CODE REVIEW

File: bsmetadata/evaluation.py [hunk 1]
```diff
@@ -348,8 +348,13 @@ def create_metadata_prompt(example: Dict[str, Any], cfg: MetadataConfig) -> str:
...
 
     # Load tokenizer
-    tokenizer = AutoTokenizer.from_pretrained(repo_args.model_name)
-    tokenizer.pad_token = tokenizer.eos_token
+    if args.untrained:
+        tokenizer = AutoTokenizer.from_pretrained(repo_args.model_name)
+        tokenizer.pad_token = tokenizer.eos_token
+    else:
+        tokenizer = AutoTokenizer.from_pretrained(
+            "bs-modeling-metadata/checkpoints_all_04_23", subfolder="tokenizer", use_auth_token=True
```
Comment (tianjianjiang) [inline bsmetadata/evaluation.py:356]: @manandey I will change this to use the arguments but thank you!

Comment (jordiclive) [reply bsmetadata/evaluation.py:356]: @tianjianjiang Are you looking at adding the HTML special tokens? Then this tokenizer would change.

**Review by jordiclive — APPROVED** [review]
lgtm. We can merge this and work on getting to the bottom of the -inf, HTML in the meantime.

File: bsmetadata/evaluation.py (hunk 1 above)
Comment (tianjianjiang) [reply bsmetadata/evaluation.py:356]: @jordiclive Yes, it's one of the reasons and yet using arguments are probably just more general but of course it doesn't matter for the current version anyway. :)
